### DISCUSSION TITLE
怎么制作一个弧形的Node?

### DISCUSSION THREAD

**Discussion by waldenwang9966** [antvis/G6#3788] [Q&A] [answered]
![Screen Shot 2022-07-11 at 10 09 24 PM](https://user-images.githubusercontent.com/68396330/178393357-a701f6e5-733e-4236-9228-68100460f7fc.jpg)
想要制作一个这样的布局，但是找了半天没有找到custom nodes可以制作出如图所示的弧形。有没有办法制作弧形节点？

**Comment by hustcc** [comment, marked as answer]
这个其实就是 cirlce 类型的节点，只不过配置为：

```ts
{
  lineWidth: 3,
  lineDash: [4, 4],
  stroke: '#fff'
}
```
